README.md

```diff
@@ -223,6 +223,8 @@ Run it:
   - Debug build is slower, but gives much more useful output in a debugger.
 - If you build a bare server you don't need to have the Irrlicht or IrrlichtMt library installed.
   - In that case use `-DIRRLICHT_INCLUDE_DIR=/some/where/irrlicht/include`.
+- IrrlichtMt can also be installed somewhere that is not a standard install path.
+  - In that case use `-DCMAKE_PREFIX_PATH=/path/to/install_prefix`.
```

Comment:
Which path? The library path? The Irrlicht clone path? What *is* the install path? Is it required to install Irrlicht?

`cmake-gui` does not expose `CMAKE_PREFIX_PATH` directly. There's `IrrlichtMt_DIR` which cannot be changed manually, but is not a hidden option.

`master` properly exposes `IRRLICHT_INCLUDE_DIR`, `IRRLICHT_LIBRARY` (and `IrrlichtMt_DIR`), which have a clear path to specify. Perhaps it's just my melting brain, or this new process does not work with the old workflow.

Reply:
Unless Irrlicht is in lib/irrlicht (relative to the top minetest directory) it will have to be installed. The install path can be absolutely any path you want; it will default to /usr. Perhaps we should have an IRRLICHT_PATH variable that CMAKE_PREFIX_PATH is initialized from.

Reply:
Unless it's in a standard path (e.g. `/usr`), `CMAKE_PREFIX_PATH` will need to be set to a folder so that `$(CMAKE_PREFIX_PATH)/lib/cmake/IrrlichtMt` exists.

> I build Minetest and Irrlicht separately without installing (RUN_IN_PLACE, basically - Windows style) and am having troubles to understand how this new system is supposed to work.

I do too and the answer to this is that this workflow will no longer work.

Reply:
>  The install path can be absolutely any path you want; it will default to /usr on Linux.

> I do too and the answer to this is that this workflow will no longer work.

Alright. I wish this were still possible to avoid an additional step while compiling, especially because it worked just fine before. Too bad that this feature is gone.
On a side note: How does this PR work on Windows? There's no such installation path for Irrlicht.

Reply:
I don't have a dev environment on Windows, which I rarely use, but Windows should have a default installation path. Do you have a source (docs preferable) for that?

Reply:
![grafik](https://user-images.githubusercontent.com/1042418/126077855-ca3af12c-b2b2-46f9-af20-1ff6f982b34f.png)

needless to say Window is not well suited to the concept of installing development files system-wide